Sources/KlaviyoSwift/KlaviyoAPI.swift

```diff
@@ -68,7 +68,7 @@ struct KlaviyoAPI {
             return .failure(.missingOrInvalidResponse(response))
         }
 
-        if httpResponse.statusCode == 429 {
+        if [429, 503].contains(httpResponse.statusCode) {
```

Comment:
Since we won't have the `Retry-After` for 503 will this retry immediately or use exponential back off?

Reply:
I believe exp backoff.

Reply:
`let nextBackoff = addJitter(to: retryAfter ?? exponentialBackOff)`

Reply:
```            let retryAfter = Int(httpResponse.value(forHTTPHeaderField: "Retry-After") ?? "0")``` but maybe it's not falsy.

Reply:
Yeah I think `nextBackoff` might be 0 in this case.